### PR TITLE
fix(dav): Use an icon with correct color for calendar user settings section

### PR DESCRIPTION
**Pull request by tcitworld**
| Before | After |
|--------|--------|
| image | image | 

Follow-up to https://github.com/nextcloud/server/pull/34746

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [ ] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [ ] Screenshots before/after for front-end changes
- [ ] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [ ] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

**Review by szaimen — APPROVED** [review]
LGTM but didnt test

File: apps/settings/lib/Sections/Personal/Calendar.php [hunk 1]
```diff
@@ -38,7 +38,7 @@ public function __construct(IL10N $l, IURLGenerator $urlGenerator) {
 	}
 
 	public function getIcon(): string {
-		return $this->urlGenerator->imagePath('core', 'caldav/time.svg');
+		return $this->urlGenerator->imagePath('dav', 'time.svg');
```
Comment (juliushaertl) [inline apps/settings/lib/Sections/Personal/Calendar.php:41]: Maybe name the file calendar.svg? Seems more fitting if it is added as a new one anyways

Comment (tcitworld) [reply apps/settings/lib/Sections/Personal/Calendar.php:41]: Done

**Review by ChristophWurst — APPROVED** [review]

**Review by juliusknorr — APPROVED** [review]

**Pull request merged**